x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/nlp/BertRequestBuilderTests.java

```diff
@@ -37,4 +39,17 @@ public void testBuildRequest() throws IOException {
         assertEquals(Arrays.asList(0, 0, 0, 0, 0), jsonDocAsMap.get("arg_2"));
         assertEquals(Arrays.asList(0, 1, 2, 3, 4), jsonDocAsMap.get("arg_3"));
     }
+
+    public void testInputTooLarge() {
```

Comment:
It would be useful to also add a test that has exactly the number of allowed tokens. Testing the boundaries!

Reply:
I would suggest also randomized input. Very Elasticsearch/Lucene style testing 😉